data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -743,5 +743,19 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_COLD", "add": 6 } ] } ]
     }
+  },
+  {
+    "id": "longsword_holy",
+    "type": "GENERIC",
+    "copy-from": "longsword",
+    "name": { "str": "holy sword" },
+    "description": "This is a summoned medieval sword, in between the lighter arming sword and later two-handed swords in size.  It glows slightly and it's edge is exceedingly sharp.",
```

Comment:
```suggestion
    "description": "This is a summoned medieval sword, between the lighter arming sword and later two-handed swords in size.  It glows slightly and its edge is exceedingly sharp.",
```

Reply:
> in between the lighter arming sword and later two-handed swords in size

This part was copied from the normal longsword